libcodechecker/cmd/cmd_line_client.py

```diff
@@ -321,11 +318,7 @@ def handle_list_results(args):
                 rows.append(
                     (checked_file, res.checkerId, sev, res.checkerMsg))
 
-        if args.output_format == 'csv':
-            writer = csv.writer(sys.stdout)
-            writer.writerows(rows)
-        else:
-            print_table(rows)
+        print(twodim_to_str(args.output_format, header, rows))
```

Comment:
`output_format` is a [_choices_](https://github.com/whisperity/CodeChecker/blob/10c8bb37b8c9a04b4fd1c0456c45a3efb15f7ad0/libcodechecker/cmd/cmd_line_client.py#L507) argument, which makes _argparse_ error if the choice is invalid.